README.md

```diff
@@ -59,7 +59,9 @@ python3 -m pip install git+https://github.com/It4innovations/ruth.git
       "speeds-path" : "",
       "travel-time-limit-perc": 0.1,
       "ptdr-path" : "",
-      "continue-from": ""
+      "continue-from": "",
+      "disable-stuck-detection": false,
```

Comment:
perhaps change to "stuck-detection", instead of disable. It may lead to mistakes.